### DISCUSSION TITLE
Adding travel points after stop

### DISCUSSION THREAD

**Discussion by rixtrix** [Courseplay/Courseplay_FS22#2291] [Ideas]
Hi it would be very nice to be able add a travel point after the stop sign. for example at the stop sign seeder folds and then goes to a place that you decide. you create you field work as normal then you open course editor and add travel points at the end

**Comment by Tensuko** [comment]
Not going to happen.
What would you want to do?
If you start your course on lanes it will stay at the field border.
So what do you want to actually do ?

**Reply by rixtrix** [reply]
A want to drive to the beginning of the field starting point (my fields are big)

**Reply by Tensuko** [reply]
That is return to start and is already on the todo list.

**Reply by rixtrix** [reply]
ok nice a found a way that's works for me but it will harvest or seed on the way a just moved the STOP to near the start point and added way points on that last way and a have a trailer ther to so it unloads and are in a good possession for next run

**Reply by S-eler** [reply]
> A want to drive to the beginning of the field starting point (my fields are big)

Use AD for it :).

**Reply by pvaiko** [reply]
Return to starting point is now implemented.
